aiofreepybox/aiofreepybox.py

```diff
@@ -63,6 +63,24 @@ def __init__(self, app_desc=app_desc, token_file=token_file, api_version='v3', t
         conn = aiohttp.TCPConnector(ssl_context=ssl_ctx)
         self._session = aiohttp.ClientSession(connector=conn)
 
+        # Detect host, port and api_version
+        r = await self._session.get('http://mafreebox.freebox.fr/api_version', timeout=self.timeout)
```

Comment:
Not sure we want to do this if values are hardcoded (not set to auto)

Reply:
You're right i also figured it was not ideal, it should be fine now.